src/libraries/System.IO.IsolatedStorage/ref/System.IO.IsolatedStorage.csproj

```diff
@@ -7,7 +7,6 @@
     <Compile Include="System.IO.IsolatedStorage.cs" />
   </ItemGroup>
   <ItemGroup>
-    <ProjectReference Include="..\..\System.IO.FileSystem\ref\System.IO.FileSystem.csproj" />
     <ProjectReference Include="..\..\System.IO.FileSystem.Primitives\ref\System.IO.FileSystem.Primitives.csproj" />
```

Comment:
You can delete `System.IO.FileSystem.Primitives.csproj` refs too while you are on it. System.IO.FileSystem.Primitives does not have anything real it in.